profile/README.md

```diff
@@ -4,9 +4,19 @@
 The LIonWeb initiative aims to facilitate the community-based development of language engineering and modeling tools on the web. 
 </strong>
 
-1. The main focus is the definition of protocols for the communication between participating software components such as repositories and clients. To make this feasible, we also define a meta-metamodel as well as a reference architecture. 
+Through the LIonWeb initiative we are aiming to:
+
+1. Create an ecosystem of interoperable components. We believe that a lively ecosystem will facilitate the adoption of language engineering and modeling solutions, as it will reduce the vendor-lock risk. It will also make possible to quickly build more advanced solutions by mixing and matching components, potentially sourced from different vendors
```

Comment:
Finish sentence with full stop.

```suggestion
1. Create an ecosystem of interoperable components. We believe that a lively ecosystem will facilitate the adoption of language engineering and modeling solutions, as it will reduce the vendor-lock risk. It will also make possible to quickly build more advanced solutions by mixing and matching components, potentially sourced from different vendors.
```